cmd/list.go

```diff
@@ -436,11 +437,66 @@ var listNotificationCmd = &cobra.Command{
 	},
 }
 
+var listProjectGroupsCmd = &cobra.Command{
+	Use:     "project-groups",
+	Aliases: []string{"pg"},
+	Short:   "List groups in a project (alias: pg)",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(lagoonCLIConfig.Current)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		if err != nil {
+			return err
+		}
+		if cmdProjectName == "" {
+			fmt.Println("Missing arguments: Project is not defined")
+			cmd.Help()
+			os.Exit(1)
+		}
+
+		current := lagoonCLIConfig.Current
+		token := lagoonCLIConfig.Lagoons[current].Token
+		lc := lclient.New(
+			lagoonCLIConfig.Lagoons[current].GraphQL,
+			lagoonCLIVersion,
+			&token,
+			debug)
+		projectGroups, err := l.GetProjectGroups(context.TODO(), cmdProjectName, lc)
+		handleError(err)
+
+		if len(projectGroups.Groups) == 0 {
+			output.RenderInfo(fmt.Sprintf("There are no projects in group '%s'", groupName), outputOptions)
```

Comment:
Where is `groupName` defined? This message is confusing. I expect this message to be like "There are no groups for project '%s'" with the cmdProjectName as substitute

Reply:
:see_no_evil: 
Will fix it up

Reply:
Based on the condition though, this shouldn't ever be reached as projects should always have 1 group. Did you encounter this error?

Reply:
Yeah I get this error message for all of our projects. We haven't assigned our projects to a group yet.

```
~ % lagoon list project-groups -p <PROJECT NAME>
Info: There are no projects in group ''
```